src/components/Profile.js

```diff
@@ -23,14 +31,38 @@ class Profile extends Component {
         .collection("Users")
         .doc(this.props.userId)
         .get();
+
       if (userData.exists) {
-        this.setState({ user: userData.data(), loading: false });
+        this.setState({ user: userData.data(), loading: false }, () =>
+          console.log("uhey", userData.id)
+        );
       } else {
         this.setState({ error: "User doesn't exist", loading: false });
       }
     } catch (err) {
       console.error("Error", err.message);
     }
+    try {
+      var recipeData = db.collection("Recipes");
+
+      var query = await recipeData
+        .where("user_id", "==", this.props.userId)
+        .get();
+
+      if (query.empty) {
+        console.log("No matching documents.");
```

Comment:
❗️  Wenn der `try` funktioniert brauchen wir die Zeile nicht mehr oder

Reply:
Ne, Firebase springt nicht in den catch, wenn das Document nicht gefunden wird, weil der Request ja erfolgreich war.